css-canvas-medical/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <!-- 1. Header Section -->
+  <div class="container">
+    <div class="row header w">
+      <a href="#" class="logo"><img src="./images/logo-blue.svg" alt="canvas"></a>
+      <nav class="nav">
+        <a href="#product" class="nav-item gray-text">Products</a>
```

Comment:
The only update you need is to create a `ul` with `li`s and have the `a` tags inside of the `li`s.